pkg/hash/hash.go

```diff
@@ -104,19 +96,25 @@ func (hash *Hash) WriteAny(data ...interface{}) (int64, error) {
 			if t.BitLen() <= params.BitsIntModN && t.Sign() == 1 {
 				t.FillBytes(b)
 			} else {
-				b, _ = t.GobEncode()
+				var err error
+				b, err = t.GobEncode()
+				if err != nil {
+					return n, fmt.Errorf("hash.Hash: GobEncode: %w", err)
+				}
```

Comment:
`GobEncode` doesn't actually return an error (even when the receiver is nil), but better safe than sorry.